src/drive/web/modules/views/OnlyOffice/Toolbar/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext } from 'react'
+
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+import { OnlyOfficeContext } from 'drive/web/modules/views/OnlyOffice'
+import { useFileWithPath } from 'drive/web/modules/views/OnlyOffice/Toolbar/useFileWithPath'
+import HomeIcon from 'drive/web/modules/views/OnlyOffice/Toolbar/HomeIcon'
+import HomeLinker from 'drive/web/modules/views/OnlyOffice/Toolbar/HomeLinker'
+import Separator from 'drive/web/modules/views/OnlyOffice/Toolbar/Separator'
+import BackButton from 'drive/web/modules/views/OnlyOffice/Toolbar/BackButton'
+import FileIcon from 'drive/web/modules/views/OnlyOffice/Toolbar/FileIcon'
+import FileName from 'drive/web/modules/views/OnlyOffice/Toolbar/FileName'
+import ReadOnly from 'drive/web/modules/views/OnlyOffice/Toolbar/ReadOnly'
+
+const Toolbar = () => {
+  const { isMobile } = useBreakpoints()
+  const { fileId, isPublic, isReadOnly } = useContext(OnlyOfficeContext)
+  const { data } = useFileWithPath(fileId)
+
+  const showBackButton = window.history.length > 1
+
+  if (!data) return null
```

Comment:
nit: Why not displaying a placeholder / loader element? Not a bing fan of having a blank component

Reply:
Idéalement il faudrait rajouter un Skeleton https://github.com/cozy/cozy-ui/issues/1604. En l'absence j'ai rajouté un spinner ici (même si le rendu vide n'est présent qu'une fraction de seconde), et surtout dans `HomeLinker` qui renvoie maintenant `children` à la place de `null` le temps d'avoir le résultat des queries. C'était plus problématique dans `HomeLinker`, qui décalait les autres éléments une fois les requêtes terminées.

Effectivement un `return null` doit être bien justifié 👍